src/internal/observable/dom/ajax.ts

```diff
@@ -5,15 +5,82 @@ import {  AjaxObservable, AjaxCreationMethod  } from './AjaxObservable';
  * It creates an observable for an Ajax request with either a request object with
  * url, headers, etc or a string for a URL.
  *
+ *
+ * ## Using ajax() to fetch the response object that is being returned from API.
+ * ```javascript
+ * import { ajax } from 'rxjs/ajax';
+ * import { map, catchError } from 'rxjs/operators';
+ * import { of } from 'rxjs';
+ *
+ * const obs$ = ajax(`https://api.github.com/users?per_page=5`).pipe(
+ *   map(userResponse => console.log('users: ', userResponse)),
+ *   catchError(error => {
+ *     console.log('error: ', error)
+ *     return of(error)
+ *   })
+ * );
+ *
+ * const subscribe = obs$.subscribe();
```

Comment:
We're not doing anything with the emitted values in any of these.

Reply:
Hey @benlesh thanks for the very quick review. Yes I see but when you open Stackblitz then consoles dont really do anything if you dont subscribe, right?